src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -4835,4 +4883,54 @@
             </Setter.Value>
         </Setter>
     </Style>
+    <Style x:Key="CheckBoxFileTrustStyle" TargetType="{x:Type CheckBox}">
+    <Setter Property="SnapsToDevicePixels" Value="true" />
+    <Setter Property="OverridesDefaultStyle" Value="true" />
+    <Setter Property="Template">
+        <Setter.Value>
+            <ControlTemplate TargetType="CheckBox">
+                <BulletDecorator VerticalAlignment="Center" 
+                                 Background="Transparent">
+                    <BulletDecorator.Bullet>
+                        <Border x:Name="Border"
+                                    Width="13"
+                                    Height="13"
+                                    Background="#373737"
+                                    BorderBrush="#666666"
```

Comment:
Can we use the color variables from DynamoColorsAndBrushes.xaml for generic ones?

Reply:
@reddyashish @zeusongit fixed in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12916/commits/6e3af78f9f27e36771c3ceb54e8ea240bf3c9047